CIP-1776/README.md

```diff
@@ -0,0 +1,84 @@
+
+Let us apply Occam's Razor, put simply “the simplest solution is almost always the best". A system of arbitrary complexity is often developed those of great intelligence - this issue plagues particle physics where increasingly complex models are proposed as a refutation to the standard model, requiring increasingly complex experimentation which ultimately results in increasingly expensive refutation of the underlying theory. This problem manifests in almost all fields of academia. 
+
+Another systemic flaw is that the ratification requirements for different governance actions are arbritrarily complex. Different actions require different combinations of approval from the Constitutional Committee, DReps, and SPOs, and have different thresholds for the percentage of active voting stake and stake held by stake pools that must be met in order for the action to be ratified. Additionally, the fallback conditions are not clear. It could be challenging for community members to keep track of these requirements and understand how to participate in the governance process effectively - dramatically reducing governance participation. 
+
+This proposal proceeds on the axiom that the flaws in CIP 1694 are self-evident (which indeed they are) and proposes a substantially more viable alternative.
+
+The core issues summarized:
+
+1. Arbritrary complexity: The proposal introduces several new concepts and mechanisms, such as the Constitutional Committee, DReps, and new types of voting certificates, which makes the system unneccisarily complex for stakeholders - this will reduce overall participation in governance.
+
+2. Poor intentive structure: The proposal mentions that DReps need to be compensated for their work, but it is not clear how this will be done or what the exact mechanisms will be. It fails to leverage the existing incentive structures of the Cardano blockchain. 
+
+3. Bootstrapping problem: The proposal relies on the presence of sufficient DReps to transition the system from its current state to decentralized governance. It is not clear how this will be achieved in practice.
+
+4. Perverse incentives: The proposal as put forward by IOG entrenches existing powers rather than truly grant control over the Cardano Blockchain to the community. 
+
+5. Exclusionary Representation: Current discussions fo CIP 1694 effectively exclude members of the developing word from access by introducing a threshold to become a DRep. 
```

Comment:
The CIP does not yet specify what this threshold would be.

Are members of the developing world excluded from staking and delegation because there's a 2ada deposit to register your staking key?  From your notes below, you require a 1ADA registration *fee* (not even a deposit).

I think a productive discussion to be had here would be to put forward what thresholds for participation you think *would* be equitable.

Obviously 1 million ADA would be highly exclusionary.  What about 500 ada? 100 ada? 10ada? 2 ada?

I think you can recognize the need for *some* kind of deposit or fee (if it was completely free, you could grind the ledger state to a halt by registering a hundred million dReps, for example).

Reply:
Hey @Quantumplation, thank you again for your feedback. As I have mentioned in my previous reply my schedule has been incredibly challenging recently - This is due to change shortly and I anticipate a definitive 'v1' of 1776 being ready for critique and review by the 1st of next month.

I actually think the issue of throughput and registration fails to consider scaling solutions for the blockchain as a whole, but as I've mentioned I'll have a more definitive statement on this at the start of next month.